src/sdl2/mouse.rs

```diff
@@ -124,84 +127,188 @@ impl MouseWheelDirection {
 }
 
 #[derive(Copy, Clone, Eq, PartialEq, Hash, Debug)]
-pub enum Mouse {
-    Left,
-    Middle,
-    Right,
-    X1,
-    X2,
-    Unknown(u8)
+pub enum Mousebutton {
```

Comment:
Can we get this in CamelCase please? The lack of capitalisation on Button is making me itchy!